composer/loggers/object_store_logger.py

```diff
@@ -301,8 +297,8 @@ def log_file_artifact(self, state: State, log_level: LogLevel, artifact_name: st
 
     def log_symlink_artifact(self, state: State, log_level: LogLevel, existing_artifact_name: str,
                              symlink_artifact_name: str, overwrite: bool):
-        """Object stores do not natively support symlinks, so we emulate symlinks by adding a .symlink file to the
-        object store, which is a text file containing the name of the object it is pointing to."""
+        # Object stores do not natively support symlinks, so we emulate symlinks by adding a .symlink file to the
+        # object store, which is a text file containing the name of the object it is pointing to.
         # Only symlink if we're logging artifact to begin with
```

Comment:
Is this meant to be commented out?

Reply:
Yes, since it's an implementation of an abstract method, it should not have a docstring.